code/src/App.js

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
-import data from './data.json';
+import React from 'react'
+import data from 'data'
+
+import Header from './Components/Header'
+import Album from './Components/Album'
+import Artist from './Components/Artist'
+import Image from './Components/Image'
+
 
 console.log(data);
 
+const albumsArray = data.albums.items 
+
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
-}
+    <> 
+      <header>
+        <Header/>
+      </header>
```

Comment:
Would it be possible to remove the html-tag if you don't use it anywhere else? Looks like you only have styled the h1 anyways 😃